extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -506,6 +506,7 @@ def to_dict(self):
         model['_c_dl'] = copy.deepcopy(self._c_dl)
         model['_c_lw'] = copy.deepcopy(self._c_lw)
         model['_c_l'] = copy.deepcopy(self._c_l)
+        model = self._make_json_serializable(model)
         return model
 
     def from_dict(self, model):
```

Comment:
Ah. The way i figured it was working was because the changes I've made to the caching part of my PR uses that method. So, I tested my classifier_services_test file and since it worked, I think this method works as well. But, I'll make sure by running a few other tests. @seanlip